src/Functions/Public/iaas/New-vRAMachineAttachedDisk.ps1

```diff
@@ -171,9 +171,9 @@
                 'ById' {
                     if ($Force -or $PsCmdlet.ShouldProcess($Id)){
                         # --- Check to see if the DiskId's were optionally present
-                        $Response = Invoke-vRARestMethod -URI "$APIUrl`/$Id`/disks" -Method GET -Body $Body
+                        $RestResponse = Invoke-vRARestMethod -URI "$APIUrl`/$Id`/disks" -Method GET -Body $Body
 
-                        CalculateOutput
+                        CalculateOutput $blockDeviceId $CompletionTimeout $WaitForCompletion $RestResponse
```

Comment:
Variable should be PascalCase

`$blockDeviceId` -> `$BlockDeviceId`

Reply:
If you look at the latest commit, this is already fixed.